pkgs/shells/fish/plugins/node-binpath.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildFishPlugin, fetchFromGitHub }:
+
+buildFishPlugin rec {
+  pname = "node-binpath";
+  version = "unversioned";
```

Comment:
"Unversioned" packages in Nixpkgs follow the format `0-unstable-YYYY-MM-DD`, where `YYYY-MM-DD` is the date of the most recent revision

Reply:
Great, was wondering about that.